install.md

```diff
@@ -2,23 +2,31 @@
 
 This file provides more detailed instructions on how to install, configure, and get the project running. Summary instructions are provided in readme.md.
 
-#Step 1 -- Activate a python environment on your local machine 
+Step 1 -- Activate a python environment on your local machine 
 
-#Step 2 - Clone the repository 
-git clone git@github.com:NOAA-OWP/t-route.git
+Step 2 - Clone the repository 
+- git clone git@github.com:NOAA-OWP/t-route.git
 
-#Step 3 - Compile necessary files
-cd t-route/src/python_routing_v02/
+Step 3 - Compile necessary files
+- cd t-route/src/python_routing_v02/
+- ./compiler.sh
 
-./compiler.sh
+Step 4 - Run the model 
 
-#Step 4 - Run the model 
+- python3 compute_nhd_routing_SingleSeg_v02.py
 
-python3 compute_nhd_routing_SingleSeg_v02.py
+Step 4 - Alternative v01 model (defaults can be overwritten with custom input yaml)
```

Comment:
@jhrehanoaa Have you confirmed this still works?

Reply:
v01 still works and accepts new changes from the cmd line and yaml. However, we no longer have a yaml file that by default works with v01.